src/components/Header/index.js

```diff
@@ -1,6 +1,5 @@
-import React, { Fragment } from 'react';
+import React from 'react';
```

Comment:
The only comment i have is that the button "move" on mouse over. This should be fixed as it's an irritant. 

![image](https://user-images.githubusercontent.com/29788342/96879364-f590ae80-1449-11eb-9970-1e1b50b559bc.png)